schema/src/main/java/org/hl7/v3/ObjectFactory.java

```diff
@@ -3509,4 +3510,9 @@ public RCMRIN030000UK06Subject createRCMRIN030000UK06Subject() {
     public JAXBElement<RCMRMT030101UK04EhrComposition> createEhrComposition(RCMRMT030101UK04EhrComposition value) {
         return new JAXBElement<RCMRMT030101UK04EhrComposition>(_EHR_COMPOSITION_QNAME, RCMRMT030101UK04EhrComposition.class, null, value);
     }
+
+    @XmlElementDecl(namespace = "", name = "RequestStatement")
+    public JAXBElement<RCMRMT030101UK04RequestStatement> createLocation(RCMRMT030101UK04RequestStatement value) {
```

Comment:
I dont think you are using this are you? Your tests are all at the ehrComposition level? If you do need it then `createRequestStatement` instead of `createLocation`(if not then delete this and the QName on line 83 please